packages/editor-ui/src/components/Node.vue

```diff
@@ -16,6 +16,11 @@
 				</n8n-tooltip>
 			</div>
 
+			<n8n-tooltip placement="top" :manual="true" :value="showWebhookNodeTooltip" popper-class="node-tooltip">
```

Comment:
one more scenario - if you move a node with tooltip
<img width="1440" alt="Screen Shot 2021-11-17 at 9 27 21 AM" src="https://user-images.githubusercontent.com/4711238/142163475-8eeb87a6-5773-4269-ad54-40a889a8cb42.png">

Reply:
This is the only thing that I cannot reproduce. The only way to reproduce it is if the user move the canvas and keep the mouse button pressed. The problem is that the canvas position is updating only if the canvas is drag&droped, and the mouse button is released.

Reply:
you can reproduce this by dragging the node while tooltip is shown

Reply:
Yup, but as I clarified in the comment above, when you drag the Node, the position of the Node is not updated anywhere in the application. The position is only updated if you drop the Node on the WF which is fine by me. Let me think for solution for it. If you have any direction how we can handle this rare situation, please let me know.

Reply:
here's a hint... this func is called when dragging starts https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/mixins/nodeBase.ts#L278

Reply:
Many thanks. Finally I see the light at the end of the tunnel :D

Reply:
@mutdmour This is fixed with the latest push.